ui/imports/utils/Global.qml

```diff
@@ -92,6 +92,8 @@ QtObject {
 
     signal privacyPolicyRequested()
 
+    signal openPaymentRequestModalRequested(var callback)
```

Comment:
The preferred way is, in general, explicit signals propagation (like in https://github.com/status-im/status-desktop/pull/16718) but I think it's ok for now.